markdown/ch8.markdown

```diff
@@ -1,6 +1,6 @@
 # Big Data in Little Laptop with Toolz
 
-> GRACIE: A knife? The guy's twelve feet tall!
+> GRACIE: A knife? The guy's twelve feet tall!  
 > JACK: Seven. Hey, don't worry, I think I can handle him.
 >
 > — Jack Burton, Big Trouble in Little China
```

Comment:
Here's the html output from `make html/ch8.html` for me:

```html
<blockquote><p>GRACIE: A knife? The guy's twelve feet tall!<br/>
JACK: Seven. Hey, don't worry, I think I can handle him.</p>
<p>— Jack Burton, Big Trouble in Little China</p>
</blockquote>
```

And the wc3 page on `br`:

> Differences Between HTML and XHTML
> In HTML, the `<br>` tag has no end tag.  
> In XHTML, the `<br>` tag must be properly closed, like this: `<br />`.

So, I think it's correctly interpreted by notedown, we might just need to change `br/` to `br`, depending on the HTMLBook requirements.